Clarify group claims / team mapping in Konnect Okta setup (#5737)
Kong/docs.konghq.com: app/konnect/org-management/okta-idp.md

{{site.konnect_short_name}}.
First, complete the following in Okta:
* [Set up an Okta application](#prepare-the-okta-application)

> * [Set up claims in Okta](#set-up-claims-in-okta) (if using group claims for team mapping)

Reviewer: ```suggestion
* [(Optional) If using group claims for team mapping, [set up claims in Okta](#set-up-claims-in-okta).
```
How about this?

> * (Optional) If using group claims for team mapping, [set up claims in Okta](#set-up-claims-in-okta).

Reviewer: ```suggestion
* (Optional) If using group claims for team mapping, [set up claims in Okta](#optional-set-up-claims-in-okta).
```

> ### Set up claims in Okta

Reviewer: ```suggestion
### (Optional) Set up claims in Okta
```

> This only needs to be followed if intending to use group claims for Konnect team mappings, this is an optional step. The connection between {{site.konnect_short_name}} and Okta uses OpenID Connect tokens. To have Okta send the correct information to your {{site.konnect_short_name}} org, set up claims to extract that information.

Reviewer: ```suggestion
If you are intending to use group claims for Konnect team mappings, follow this guide to set them up. Otherwise, skip to [Add a user to your application](#add-a-user-to-your-application).

The connection between {{site.konnect_short_name}} and Okta uses OpenID Connect tokens. To have Okta send the correct information to your {{site.konnect_short_name}} org, set up claims to extract that information.
```
